### ISSUE TITLE
Add link to download invoice from payment list on the Edit Member page 

### ISSUE THREAD

**Issue by thomasfw** [restrictcontentpro/restrict-content-pro#819]
It can be hard to find a payment and download an invoice for a specific transaction (especially as the _Payments_ search only works on username/email, and not first/last name - related #650).

The list of payments on the _Edit Member_ page helps, but there's no invoice link. We can click through to the payment under _Transaction ID_, but no invoice link there either.

Cheers

**Comment by nosegraze** [comment]
PR: #1172

How about this for now on the Edit Payment page?

![rcp-view-invoice](https://cloud.githubusercontent.com/assets/6324272/25006236/12f48c3c-2054-11e7-898e-35e24db4372b.png)

I'm not sure about adding it to the table since the table is already quite large and we have plans to redesign it soon anyway ( #1003 ).

**Comment by mindctrl** [comment]
On the payment table at the bottom of edit member page, do we need the Subscription Key column for anything? It's the widest column in the table and isn't actionable or really all that useful.

**Comment by nosegraze** [comment]
I can't imagine we'd need it.

**Comment by mindctrl** [comment]
I agree. I vote we ditch it and move View Invoice to the table as a link in the last column.

**Comment by nosegraze** [comment]
Updated.
